app/assets/stylesheets/config/_bootstrap_variables.scss

```diff
@@ -14,7 +14,7 @@ $body-color: $gray;
 $primary:    #10E380;
 $success:    $green;
 $info:       #EE4266;
-$danger:     $red;
```

Comment:
The same here question here @gitwithuli . Will the danger color set to green affect someone's else css code?